include/seqan3/alphabet/adaptation/char.hpp

```diff
@@ -33,10 +33,8 @@ namespace seqan3::detail
 //!\brief Whether a type is `char`, `char16_t`, `char32_t` or `wchar_t` (type trait).
 //!\ingroup adaptation
 template <typename type>
-constexpr bool is_char_adaptation_v = std::same_as<type, char>     ||
-                                      std::same_as<type, char16_t> ||
-                                      std::same_as<type, char32_t> ||
-                                      std::same_as<type, wchar_t>;
+constexpr bool is_char_adaptation_v = std::same_as<type, char> || std::same_as<type, char16_t> ||
+    std::same_as<type, char32_t> || std::same_as<type, wchar_t>;
```

Comment:
this change seems unrelated

Reply:
Resolves: "weird spacing in is_char_adaptation_v, is_uint_adaptation_v"

Reply:
hideinitializer